### PR TITLE
Chore/integ 1426 adjust landing and ref content

### PR DESCRIPTION
**Pull request by alanag13**
this merges some changes that were in [Tony's branch](https://github.com/code42/developer.code42.com/blob/tschwandt-landing-page/source/index.html.md.erb) with some content/recommendations that Rob supplied that is discussed in [INTEG-1426](https://jira.corp.code42.com/browse/INTEG-1426) (internal link)

### CODE REVIEW

File: source/api/getting_started.rmd [hunk 1]
```diff
@@ -66,94 +42,18 @@ A successful request returns an authentication token. For example:
...
-    "maintenanceMode": false,
-    "tenantUid": "42b42dfe-4242-4242-428a-8a1dfd352f2f",
-    "masterServicesAgreement": {
-      "accepted": true,
-      "acceptanceRequired": false
-    }
-  },
-  "error": null,
-  "warnings": null
+    "name": "Example customer",
```
Comment (alanag13) [inline source/api/getting_started.rmd:55]: this was response was still an example from the old, non-gateway api, so i fixed it.

File: source/stylesheets/redocoverrides.css.scss [hunk 2]
```diff
@@ -49,3 +49,7 @@ h2 { color: #333333 !important;}
         display: none;
     }
 }
+
+div[data-section-id] {
```
Comment (alanag13) [inline source/stylesheets/redocoverrides.css.scss:53]: the default styles for redoc add an absurd amount of padding between headings, so i fixed that.

File: source/api/getting_started.rmd [hunk 3]
```diff
@@ -1,45 +1,21 @@
...
-
-* API resources themselves only work for you under these conditions:
-  * You have a [product plan](https://support.code42.com/Terms_and_conditions/Code42_customer_support_resources/Code42_product_plans) that includes access to the Code42 API.
-  * Your credentials rely on local authentication. SSO or authentication through any third-party provider will not work.
-  * Your [role provides permission](https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Manage_user_roles) to access the data necessary to a given API resource. If your API calls fail because you do not have permission to use them, you will see reply messages like these: `HTTP 401 Unauthorized`, `HTTP 401 Could not authenticate user`, `Your Code42 product plan does not permit use of the Code42 API`.
-  * The examples in the articles here use curl.
-
----
-
 ## Request URLs
```
Comment (unparalleled-js) [inline source/api/getting_started.rmd:2]: For md document purposes, could add a newline between the headers

File: source/index.html.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+title: Home
+
+search: true
+
+code_clipboard: true
+
+hide_toc: true
+---
+
+# Code42 Developer Portal<br>Automate and integrate with Incydr
```
Comment (unparalleled-js) [inline source/index.html.md:11]: If there is a way to achieve this without using html, that would be preferred

File: source/api/getting_started.rmd [hunk 5]
```diff
@@ -1,45 +1,21 @@
...
-
-* United States:
-
-  * If you sign in to the Code42 console at [https://www.crashplan.com/console](https://www.crashplan.com/console), use `https://api.prod.ffs.us2.code42.com/<resource>`.
-
-  * If you sign in to the Code42 console at [https://console.us.code42.com/console](https://console.us.code42.com/console), use `https://api.us.code42.com/<resource>`.
+The domain used for making API requests can be determined using the domain you use to log in to the Code42 console. All API requests should be made using https.
 
-* Ireland: If you sign in to the Code42 console at [https://console.ie.code42.com/console](https://console.ie.code42.com/console), use `https://api.ie.code42/<resource>`.
+| Console Domain | API Domain |
```
Comment (unparalleled-js) [inline source/api/getting_started.rmd:6]: This table could have equal column widths

**Review by unparalleled-js — APPROVED** [review]

File: source/api/getting_started.rmd [hunk 6]
```diff
@@ -1,45 +1,21 @@
...
-curl -X GET -u "username" -H "Accept: application/json" https://api.us.code42.com/v1/auth/
-```
+<br>
 
 ## Authentication
 
 Most API calls require an [authentication token](https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Code42_API_resources/Code42_API_authentication_methods#Use_token_authentication) in the header of the request. To obtain an authentication token, use your Code42 administrator credentials to submit a `GET` request to the `/v1/auth/` API. (For details, see the [Auth API documentation](/api/#tag/Auth).)
 
-In the following example, replace `<Code42Username>` with a Code42 administrator username, and replace `<RequestURL>` with the [request URL](#request-urls) of your Code42 cloud instance:
+In the following example, replace `<Code42Username>` with a Code42 administrator username, and replace `<RequestURL>` with the [request URL](/api/#section/Getting-started/Request-URLs) of your Code42 cloud instance:
```
Comment (unparalleled-js) [inline source/api/getting_started.rmd:18]: Now that the scheme is no longer in the  request URL examples, this line makes it sound like no scheme is needed.

**Review by adelegard — APPROVED** [review]

File: source/index.html.md (hunk 4 above)
Comment (alanag13) [reply source/index.html.md:11]: Not a "clean" one -- technically I could add another h1 heading and then add some gross css to decrease the margin / padding on it or something, but I don't think that's better

Comment (unparalleled-js) [reply source/index.html.md:11]: K, no prob, I agree this is better

**Comment by unparalleled-js** [comment]
Do we want to add the scheme to all the examples? Doesnt have to be in this PR. Or maybe just "Replace RequestUrl with https://<request-url>

**Review by unparalleled-js — DISMISSED** [review]

**Review by unparalleled-js — APPROVED** [review]

**Pull request merged**